modules/datadog-monitor/catalog/monitors/lambda.yaml

```diff
@@ -0,0 +1,22 @@
+# The official Datadog API documentation with available query parameters & alert types:
+# https://docs.datadoghq.com/api/v1/monitors/#create-a-monitor
+
+lambda-errors:
+  name: AWS Lambda ${ tenant }-${ environment }-${ stage } has errors
```

Comment:
The lambda-log-forwarder monitor needs the same fully qualified name. name: "(Lambda) ${ tenant }-${ environment }-${ stage } - Datadog Lambda Forwarder Config Changed"

https://github.com/cloudposse/terraform-aws-components/blob/f7a75efa0585257556f1ce36bd547afed4fb714a/modules/datadog-monitor/catalog/monitors/lambda-log-forwarder.yaml#L5